modules/claims_api/app/workers/claims_api/flash_updater.rb

```diff
@@ -4,11 +4,14 @@
 require 'sentry_logging'
 
 module ClaimsApi
-  class FlashUpdater
+  class FlashUpdater < EncryptPii
     include Sidekiq::Worker
     include SentryLogging
 
-    def perform(user, flashes, auto_claim_id = nil)
+    def perform(flashes, auto_claim_id = nil)
```

Comment:
can the claim id's be `nil` here? curious if that could ever happen. The first line of the job that kicks these off is: 
`auto_claim = ClaimsApi::AutoEstablishedClaim.find(auto_claim_id)`